app/helpers/dashboard_helper.rb

```diff
@@ -54,14 +54,14 @@ def count_reports(hosts, options = {})
   end
 
   def get_overview_json(report, options = {})
-    [
+    { data: [
       [_('Active'), report[:active_hosts_ok_enabled], report_color[:active_hosts_ok_enabled]],
       [_('Error'), report[:bad_hosts_enabled], report_color[:bad_hosts_enabled]],
       [_('OK'), report[:ok_hosts_enabled], report_color[:ok_hosts_enabled]],
       [_('Pending changes'), report[:pending_hosts_enabled], report_color[:pending_hosts_enabled]],
       [_('Out of sync'), report[:out_of_sync_hosts_enabled], report_color[:out_of_sync_hosts_enabled]],
       [_('No report'), report[:reports_missing], report_color[:reports_missing]],
-    ].to_json
+    ]}.to_json
```

Comment:
I believe we don't need to encode to json now. Maybe arguably we could rename the method then.
```suggestion
    ]}
```